_episodes/07-visualization-ggplot-python.md

```diff
@@ -30,6 +30,12 @@ and [The Grammar of Graphics][grammar-of-graphics]
 by Leland Wilkinson. The `plotnine`
 package is built on top of Matplotlib and interacts well with Pandas.
 
+> ## Reminder
+> `plotnine` is not included in the standard Anaconda installation and needs
+> to be installed separately. If you haven't done so already, you can find installation
+> instructions on the Setup page: <https://datacarpentry.org/python-ecology-lesson/setup.html#download-plotting-package>.
```

Comment:
```suggestion
> instructions on the [Setup page][setup-plotnine].
```

Reply:
I suggest creating `lesson-setup-plotnine` link at the end of this document (line 601) like so:

```markdown
[setup-plotnine]: https://datacarpentry.org/python-ecology-lesson/setup.html#download-plotting-package
```